controller/utils/Logger.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+
+class Logger(object):
+
+    def __init__(
+        self,
+        debug=False,
+        logger=None
+    ):
+        self._set_logger(logger, debug)
+
+    def debug(self, message):
+        self.logger.debug(message)
+
+    def info(self, message):
+        self.logger.info(message)
+
+    def error(self, message):
+        self.logger.error(message)
+
+    def _set_logger(self, logger, debug):
+        """
+        Instance the class logger.
+
+        :param logger: The logging object. CONTROLLER if None
+            logger is provided.
+        :param debug: True/False to activate/deactivate debug logger.
+        """
+        if isinstance(logger, logging.Logger):
+            self.logger = logger
+        else:
+            if isinstance(logger, str):
+                self.logger = logging.getLogger(logger)
+            else:
+                self.logger = logging.getLogger('CONTROLLER')
+
+            if not self.logger.hasHandlers():
+                l_handler = logging.StreamHandler()
+                l_format = '[%(asctime)s][%(name)s][%(levelname)s] %(message)s'
+                l_format = logging.Formatter(l_format)
+                l_handler.setFormatter(l_format)
+                self.logger.addHandler(l_handler)
+
+        if debug:
+            self.logger.setLevel(logging.DEBUG)
+        else:
+            self.logger.setLevel(logging.INFO)
```

Comment:
Add a variable (global) `logger` that give access to all the process to make log without creating a logger